includes/Core/Admin/Dashboard.php

```diff
@@ -81,19 +81,6 @@ function () {
 				$this->add_widgets();
 			}
 		);
-
-		$wp_dashboard_callback = function() {
-			if ( 'dashboard' === get_current_screen()->id && current_user_can( Permissions::VIEW_DASHBOARD ) ) {
-				// Enqueue styles.
-				$this->assets->enqueue_asset( 'googlesitekit-wp-dashboard-css' );
-
-				// Enqueue scripts.
-				$this->assets->enqueue_asset( 'googlesitekit-wp-dashboard' );
-				$this->modules->enqueue_assets();
-			}
-		};
-
-		add_action( 'admin_enqueue_scripts', $wp_dashboard_callback, 30 );
```

Comment:
This part is unnecessarily hooked into `admin_enqueue_scripts` and requires we check the cap in both places. `wp_dashboard_setup` is already called on the dashboard only and other core widgets (e.g. site health) also enqueue assets here before registering the widget so this can be simplified by moving below.

Reply:
Nice!